tarantool/__init__.py

```diff
@@ -76,3 +80,8 @@ def connectmesh(addrs=({'host': 'localhost', 'port': 3301},), user=None,
 __all__ = ['connect', 'Connection', 'connectmesh', 'MeshConnection', 'Schema',
            'Error', 'DatabaseError', 'NetworkError', 'NetworkWarning',
            'SchemaError', 'dbapi']
+
+# ConnectionPool is supported only for Python 3.7 or newer.
```

Comment:
Can you highlight it for me please why connectionPool is supported only for Python 3.7 or newer?

Reply:
I used [dataclasses](https://docs.python.org/3/library/dataclasses.html) in ConnectionPool code. They were introduced in Python 3.7. It would be possible to write the same code without dataclasses, but it would be less convenient, and since
* Python 3.6 [ended its life in 2021](https://docs.python.org/3/library/dataclasses.html);
* it affects only ConnectionPool and all other old features are available for Python 3.x;
we decided to support only 3.7 here.

By the way, the main reason for dropping Python 2 was our implementation of connection interface (we use metaclasses).